### DISCUSSION TITLE
Extracting the github repo name for prBodyNotes display

### DISCUSSION THREAD

**Discussion by elmaso1620** [renovatebot/renovate#13601] [Request Help]
I'm trying to set a prBodyNotes value so that my PR can display the dependency repo name.

For example, given the sourceUrl config variable exists with a value https://githhub.com/renovatebot/renovate, how would I get the 'renovatebot' bit?

Alternatively, there is a sourceRepoSlug config variable like 'renovatebot-renovate', can the first element be extracted when defining a prBodyNotes string?

It seems that you can even use 'if' statements during the prBodyNotes definition e.g. https://docs.renovatebot.com/configuration-options/#prbodynotes Is there any documentation around this area? Can a regex be used somehow?

**Comment by rarkins** [comment]
The variable you're after is not currently available, and neither is regex, but you're welcome to submit a PR which generates `sourceRepoOrg` (e.g. `renovatebot` from `renovatebot/renovate`) and `sourceRepoName` (e.g. `renovate` from `renovatebot/renovate`).

**Reply by elmaso1620** [reply]
Be gentle, it's my first! https://github.com/renovatebot/renovate/pull/13620

**Reply by astellingwerf** [reply]
I can see a use case where these properties could support easier customization of the onboarding PR in large organizations. If I can set the `onboardingConfig` with a `matchSourceRepoOrgName`,  `matchSourceRepoOrgPattern` , or `matchSourceRepoOrgPrefix` property, it would prevent the need for every newly discovered project's Onboarding PR in an org to be manually customized in the same way.
